atlasdb-config/src/main/java/com/palantir/atlasdb/http/FailoverFeignTarget.java

```
@@ -56,8 +59,13 @@
     final ThreadLocal<Integer> mostRecentServerIndex = new ThreadLocal<>();
 
     public FailoverFeignTarget(Collection<String> servers, Class<T> type) {
+        this(servers, DEFAULT_MAX_BACKOFF_MILLIS, type);
+    }
+
+    public FailoverFeignTarget(Collection<String> servers, int maxBackoffMillis, Class<T> type) {
         this.servers = ImmutableList.copyOf(ImmutableSet.copyOf(servers));
         this.type = type;
+        this.maxBackoffMillis = maxBackoffMillis;
```

We should fail fast if this is 0 or below